support-workers/src/main/scala/com/gu/support/workers/lambdas/CreatePaymentMethod.scala

```diff
@@ -47,7 +47,7 @@ class CreatePaymentMethod(servicesProvider: ServiceProvider = ServiceProvider)
   ): Future[PaymentMethod] =
     state.paymentFields match {
       case stripeHosted: StripeHostedPaymentFields =>
-        createStripeHostedPaymentMethod(stripeHosted, services.stripeService)
+        createStripeHostedPaymentMethod(stripeHosted, services.stripeService, state.user)
```

Comment:
The user is passed here because it contains everything we need to generate the hash.